homeassistant/components/esphome/__init__.py

```diff
@@ -284,6 +285,30 @@ async def send_home_assistant_state_event(event: Event) -> None:
             _send_home_assistant_state(entity_id, attribute, hass.states.get(entity_id))
         )
 
+    pipeline_run: VoiceAssistantUDPServer | None = None
+
+    async def handle_pipeline_start() -> int:
+        """Start a voice assistant pipeline."""
+        nonlocal pipeline_run
+
+        if pipeline_run is not None:
+            return
+
+        pipeline_run = VoiceAssistantUDPServer(hass, entry_data)
```

Comment:
Can we rename this variable name. It's a UDP server and we don't want to confuse it with  it being an instance of the class `PipelineRun`